SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -34,17 +34,34 @@
 #include <numeric>
 #include <vector>
 
+#include "../../include/fluid/CConstantConductivity.hpp"
+#include "../../include/fluid/CConstantConductivityRANS.hpp"
+#include "../../include/fluid/CConstantPrandtl.hpp"
+#include "../../include/fluid/CConstantPrandtlRANS.hpp"
+#include "../../include/fluid/CConstantViscosity.hpp"
+#include "../../include/fluid/CPolynomialConductivity.hpp"
+#include "../../include/fluid/CPolynomialConductivityRANS.hpp"
+#include "../../include/fluid/CPolynomialViscosity.hpp"
+#include "../../include/fluid/CSutherland.hpp"
+
 CFluidScalar::CFluidScalar(su2double val_Cp, su2double val_gas_constant, const su2double value_pressure_operating,
                            CConfig* config)
     : CFluidModel() {
   n_species_mixture = config->GetnSpecies() + 1;
```

Comment:
Why do we add an additional species here?  Also could this be const?

Reply:
Hi, it is because in this class the config pointer is available, before the constructor of CFluidScalar, this pointer is not available and I am still figuring out how to make n_species_mixture constant.